src/compiler/diagnosticMessages.json

```diff
@@ -7973,10 +7965,6 @@
         "category": "Error",
         "code": 18051
     },
-    "Non-abstract class '{0}' does not implement all abstract members of '{1}'": {

```

Comment:
Sorry, I wasn’t suggesting removing these—I think the “Non-abstract” language is important—I just wanted to avoid the error message structure @typeholes pointed out:

> ```
> Non-abstract class 'C' does not implement inherited abstract members 'bm1', 'bm2', 'bm3', 'm1', 'm2', 'm3' from class 'B'.ts(2654)
> ```

However, it looks like this message was never possible. I’m fine with the messages the way they were, or this one could be rewritten as

```
Non-abstract class '{0}' is missing implementations for the following members of '{1}: {2}.'
```

But I only suggested changes because I thought it was possible for the name of the abstract class to be buried behind a long list. Sorry for the confusion.